force-app/main/default/applications/Mainly_Digital.app-meta.xml

```diff
@@ -14,6 +14,7 @@
     <navType>Standard</navType>
     <tabs>Template__c</tabs>
     <tabs>Journey__c</tabs>
+    <tabs>Create_Journey_from_Template</tabs>
```

Comment:
If you can delete this reference, it will cause problems with your deployment in the future.

Reply:
Deleted.